lom/src/plugins/vendors/generic/GenericPluginMitigation.go

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
This is not a generic one, but SONiC specific.

Reply:
added comment